migrations/000019_permanent_id_request_status_codes.up.sql

```diff
@@ -0,0 +1,19 @@
+BEGIN;
+
+SET search_path = public, pg_catalog;
+
+--
+-- The statuses that have been applied to each Permanent ID request.
+--
+CREATE TABLE IF NOT EXISTS permanent_id_request_status_codes (
```

Comment:
Should this table's initial data (now in `old/main/data/01_permanent_id_request_status_codes.sql`) be added by this migration?
https://github.com/cyverse-de/metadata-db/blob/9907bd989e0d0328ac22f565a7981a8ae99e08b1/src/main/data/01_permanent_id_request_status_codes.sql#L4-L16